csharp/src/Apache.Arrow.Flight/Server/FlightServer.cs

```diff
@@ -48,9 +48,10 @@ public virtual Task DoAction(FlightAction request, IAsyncStreamWriter<FlightResu
             throw new NotImplementedException();
         }
 
-        public virtual Task<Schema> GetSchema(FlightDescriptor request, ServerCallContext context)
+        public virtual async Task<Schema> GetSchema(FlightDescriptor request, ServerCallContext context)
         {
-            throw new NotImplementedException();
+            var info = await GetFlightInfo(request, context);
+            return info.Schema;
```

Comment:
This change does not seems particular to the example, so maybe resolve in a different issue/PR. You can also include the other non-example changes alongside the new PR.

Reply:
Good point. Was trying to keep track of usability issues I found while writing examples. I'll resubmit this change in another PR.